### PR TITLE
FOUNDATIONS: Update Group

### PR DESCRIPTION
**Pull request by eriadhami**
Closes Issue no. #204

### CODE REVIEW

**Comment by hassanhabib** [comment]
@eriadhami we should change this to `Modify` instead of `Update`.
@ShriHumrudha please take a look at this PR.

**Comment by eriadhami** [comment]
@hassanhabib I was thinking that for social networks Update was more accurate for business language that's why I didn't use Modify.

**Comment by hassanhabib** [comment]
@eriadhami please resolve conflicts here.

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
```
Comment (ShriHumrudha) [inline .vscode/launch.json:1]: Local files should not be checked-in

File: Taarafo.Core.Tests.Unit/Services/Foundations/Groups/GroupServiceTests.Exceptions.Modify.cs [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+            this.dateTimeBrokerMock.VerifyNoOtherCalls();
+            this.storageBrokerMock.VerifyNoOtherCalls();
+            this.loggingBrokerMock.VerifyNoOtherCalls();
+        }
+
+        [Fact]
+        public async void ShouldThrowValidationExceptionOnUpdateIfReferenceErrorOccursAndLogItAsync()
+        {
+            // given
+            Group someGroup =
```
Comment (ShriHumrudha) [inline Taarafo.Core.Tests.Unit/Services/Foundations/Groups/GroupServiceTests.Exceptions.Modify.cs:71]: Consider moving all the lines till 81 to a single line code back to back. I do not see much troubling pattern if they are on the same line.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]
I see both Update and Add PRs here.

**Pull request closed without merging**